src/golang/lib/workflow/operator/response.go

```diff
@@ -0,0 +1,60 @@
+package operator
+
+import (
+	"github.com/aqueducthq/aqueduct/lib/collections/operator"
+	"github.com/aqueducthq/aqueduct/lib/collections/operator_result"
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	"github.com/google/uuid"
+)
+
+type Response struct {
+	Id          uuid.UUID      `json:"id"`
+	Name        string         `json:"name"`
+	Description string         `json:"description"`
+	Spec        *operator.Spec `json:"spec"`
+
+	Inputs  []uuid.UUID `json:"inputs"`
+	Outputs []uuid.UUID `json:"outputs"`
+}
+
+type RawResultResponse struct {
```

Comment:
Can we add a comment inside each of these `Raw` structs - what are they supposed to be mirroring what is in the database? It's not super intuitive what this "Raw" means.